pyproject.toml

```diff
@@ -1,2 +1,3 @@
 [build-system]
-requires = ["setuptools>=30.3", "wheel", "setuptools_scm>=1.15"]
+requires = ["setuptools>=34.4", "wheel", "setuptools_scm>=1.15"]
```

Comment:
I'm not sure whether we can rely on this solely with the currently default pip versions available to users.

Reply:
It's not really reliant on it (yet). It still supplies 'setup_requires'. But still, only developers of cheroot are likely to be installing with something besides pip, so they'll all get wheels.

Reply:
At @ansible we see so much weird retro envs so I just stopped being optimistic about this at some point...